source/treeInterceptorHandler.py

```diff
@@ -134,6 +144,8 @@ def _set_passThrough(self, state):
 		if self._passThrough == state:
 			return
 		self._passThrough = state
+		browseMode = not self._passThrough

```

Comment:
why does this use the underscored `_passThrough` but the changes in `api.py` uses `self.passThrough`